vkHW/vkHW/Views/News/NewsTableViewController.swift

```diff
@@ -8,40 +8,26 @@ final class NewsTableViewController: UITableViewController {
     // MARK: Constants
 
     private enum Constants {
-        static let cellIdentifier = "NewsTableViewCell"
-        static let imageNameText = "0"
-        static let postText = "suiiii"
-        static let personalNameText = "oleg"
+        static let newsHeaderCellIdentifier = "NewsHeaderCell"
+        static let newsFooterCellIdentifier = "NewsFooterCell"
+        static let newsPostCellIdentifier = "NewsPostCell"
+        static let newsImageCellIdentifier = "NewsImageCell"
+    }
+
+    private enum NewsCellTypes: Int, CaseIterable {
+        case header
+        case content
+        case footer
     }
 
     // MARK: Private Properties
 
-    private var posts = [
-        Post(
-            personImageName: Constants.imageNameText,
-            personName: Constants.personalNameText,
-            postText: Constants.postText,
-            postImageName: Constants.imageNameText
-        ),
-        Post(
-            personImageName: Constants.imageNameText,
-            personName: Constants.personalNameText,
-            postText: Constants.postText,
-            postImageName: Constants.imageNameText
-        ),
-        Post(
-            personImageName: Constants.imageNameText,
-            personName: Constants.personalNameText,
-            postText: Constants.postText,
-            postImageName: Constants.imageNameText
-        ),
-        Post(
-            personImageName: Constants.imageNameText,
-            personName: Constants.personalNameText,
-            postText: Constants.postText,
-            postImageName: Constants.imageNameText
-        )
-    ]
+    private var posts: [Posts] = []
+    private var news: [NewsFeed] = []
+
+    // MARK: - Public Properties
+
+    private var networkService = NetworkService()
```

Comment:
```suggestion
    private var networkService = NetworkService()
```
В другую марку, это ж приватное свойство.